Tools/Frame_params/Sub/bluerov2-4_0_0.params

```diff
@@ -410,8 +410,8 @@ PSC_ANGLE_MAX,0.0
 PSC_POSXY_P,1.0
 PSC_POSZ_P,3.0
 PSC_VELXY_D,0.0
-PSC_VELXY_D_FILT,5.0
```

Comment:
We shouldn't change the bluerov-4.0 parameter names because these are for use with 4.0 that will use the old param names.